third_party/maya/lib/usdMaya/MayaTransformWriter.h

```diff
@@ -64,6 +63,7 @@ class MayaTransformWriter : public MayaPrimWriter
 
     PXRUSDMAYA_API
     MayaTransformWriter(MDagPath & iDag, UsdStageRefPtr stage, const JobExportArgs & iArgs);
```

Comment:
Doesn't look like the implementation for this constructor exists anymore. (remove declaration since looks like all references were switched to the new constructor?)